netebpfext/user/fwp_um.cpp

```diff
@@ -682,8 +765,10 @@ _IRQL_requires_max_(DISPATCH_LEVEL) void NTAPI
     FwpsApplyModifiedLayerData0(_In_ UINT64 classifyHandle, _In_ void* modifiedLayerData, _In_ UINT32 flags)
 {
     UNREFERENCED_PARAMETER(classifyHandle);
-    UNREFERENCED_PARAMETER(modifiedLayerData);
     UNREFERENCED_PARAMETER(flags);
+    UNREFERENCED_PARAMETER(modifiedLayerData);
```

Comment:
I think this line should not be needed.  On a release build, ebpf_assert in 771 is defined as 
```
#define ebpf_assert(x) (void)(x)
```
which means `UNREFERENCED_PARAMETER` is redundant with that.

Reply:
Fixed.